### DISCUSSION TITLE
Include tablename in returned data?

### DISCUSSION THREAD

**Discussion by cloudorbush** [supabase/supabase#4974] [Questions]
It may sound like a strange inquiry, but here's my use case:

I'm building an activity feed where I have to merge data from multiple tables (e.g. posts, comments, likes, etc...) and sort them according to a timestamp.

To display the right component I need to know from which table the data was fetched.

I guess one way to do this would be by manually adding a `type` column to every table in the database, but this seems redundant.

So my question is, is there an easy way to see in the data which table it came from?

Any thoughts are welcome.

**Comment by GaryAustin1** [comment]
You might want to consider adding some more detail about what your "feed" looks like at either the api or client level.  
It would seem you likely have control over the column names so would be able to differentiate on the presence of certain field names in the result.

**Reply by cloudorbush** [reply]
Would you recommend adding that extra detail to the client or the database?

**Reply by GaryAustin1** [reply]
No, you missed my point... you need to give a better explanation of what you are doing.  Show some example data in the feed or what you want.  It is too hard to tell your issue for me, or likely others to help you, especially when you can name columns and know what type of table they came from.
